R/with.R

```diff
@@ -8,7 +8,12 @@
 #' @param defaults named list of elements to modify.
 #' @return A modified list of elements, sorted by name. To achieve this sort in a platform-independent way, two
 #'   transformations are applied to the names: (1) replace `_` with `0` and (2) convert [tolower()].
-#' @seealso [linters_with_tags], [linters_with_defaults] for creating linter lists.
+#' @seealso
+#' - [linters_with_defaults] for basing off lintr's set of default linters.
```

Comment:
linters_with_tags was removed here.

Reply:
Added it back.

Also added `@family` tag to make sure all related functions here are interlinked.